lib/rucio/web/rest/flaskapi/v1/redirect.py

```diff
@@ -141,17 +142,7 @@ def generate():
                     yield '  <glfn name="/atlas/rucio/%s:%s">' % (rfile['scope'], rfile['name'])
                     yield '</glfn>\n'
 
-                    # sort the actual replicas if necessary
-                    if select == 'geoip':
-                        replicas = sort_geoip(dictreplica, client_location['ip'], ignore_error=True)
-                    elif select == 'closeness':
-                        replicas = sort_closeness(dictreplica, client_location)
-                    elif select == 'dynamic':
-                        replicas = sort_dynamic(dictreplica, client_location)
-                    elif select == 'ranking':
-                        replicas = sort_ranking(dictreplica, client_location)
-                    else:
-                        replicas = sort_random(dictreplica)
+                    replicas = sort_replicas(dictreplica, client_location, selection=sortby, default='random')
```

Comment:
default=geoip?

Reply:
no, the redirect endpoint is out of scope.